tools/molecule.spec

```diff
@@ -0,0 +1,59 @@
+%global srcname molecule
```

Comment:
I saw several projects using `packaging/rpm/` dir as a convention for the spec file. Can we use the same location?